phpunit.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phpunit bootstrap="tests/config/bootstrap.php" colors="true" cacheResult="false">
```

Comment:
Could likely reference `vendor/autoload.php` directly rather than the small bootstrap.php file. Save having to have the tests/config folder.

Reply:
Good call. It's not terribly likely we'll need to load anything else in the bootstrap.